website/sidebars.js

```diff
@@ -116,7 +116,10 @@ const sidebars = {
     "concepts/concepts-index",
     {
       "Service Overviews": [
+        "concepts/filter_service/filter-overview",
+        "concepts/gps_service/gps-overview",
```

Comment:
These should probably go roughly in complexity / dependency order
```suggestion
        "concepts/gps_service/gps-overview",
        "concepts/filter_service/filter-overview",
```

Reply:
I agree. I wasn't sure if we should organize it by alphabetical order or complexity.